test/ci_integration.sh

```diff
@@ -28,8 +30,7 @@ finish() {
   # take care of the zone entries (which will produce an error), and a second
   # time to get rid of the zone.
   set +e
-  bundle exec kitchen destroy
-  bundle exec kitchen destroy
+  kitchen destroy
```

Comment:
Have we confirmed that removing the double-`destroy` has the intended behavior? Historically, when this has been in place, it's been because of an upstream bug or similar.

Reply:
AFAIK bug was fixed

Reply:
Fixed